docs/source/API/core/Macros.rst

```diff
@@ -134,6 +134,8 @@ These defines give information about what third-party libaries Kokkos was compil
 +-------------------------------+-----------------------------------------------------------------------------------------------------------------------+
 | ``KOKKOS_ENABLE_LIBQUADMATH`` | Defined if Kokkos links to the `GCC Quad-Precision Math Library API <https://gcc.gnu.org/onlinedocs/libquadmath/>`_.  |
 +-------------------------------+-----------------------------------------------------------------------------------------------------------------------+
+| ``KOKKOS_ENABLE_ONEDPL``      | Defined if Kokkos links to the `oneDPL library <https://github.com/oneapi-src/oneDPL>`_ when using the SYCL backend.  |
++-------------------------------+-----------------------------------------------------------------------------------------------------------------------+
```

Comment:
Should we expand to say what it is used for and that things like `Kokkos::sort` might be less performant if not found?

It is fine if you want to do as a follow up.

Reply:
[`d02c302` (#496)](https://github.com/kokkos/kokkos-core-wiki/pull/496/commits/d02c302d8856c12976718bca2e5396f95a48c848)